website/docs/general/blog-contributing-guide.md

```diff
@@ -91,274 +105,94 @@ keywords:
-- **Technology**: Technical articles, easily confused with Practical Case. The factors that determine whether an article is a Technology or a Practical Case are: the content of the article and the subject of the description. For example, if the article is about "Running Apache APISIX on xxx platform", then it is a Practical Case; for example, if the article is about technical stuff, "Apache APISIX v.s. Envoy", then it is Technology.
-- **User Case**: User Case, this is also very straightforward. Please tell us how you use Apache APISIX and your feelings about Apache APISIX.
+Each post can have multiple tags. The tags used currently are given below and each post usually fits into one or more of these tags.
 
-We can help authors to use a better category once a pull request is submitted.
+Each post can have more than one tag. The available tags and explanations are as follows. If none of the tags below fits, please leave a comment in your pull request, and we will handle it together. Please note that these tags and rules of applying tags could change over time.
 
-#### How to get the authors.image_url
+- **Community**: Everything related to community, for example, "How to contribute to an open source project without writing code?".
+- **Events**: Related to events, for example, live streams, event previews, meetups and project meetings.
+- **Interview**: For example, Dr. Yang Li interview, Summer of Programming interview.
+- **Practical Case**: Includes best practices to follow. This is easily confused with **Technology**. The content of the article determines which tag the post belongs to. For example, "Running Apache APISIX on xxx platform" would belong to the Practical Case tag and "Apache APISIX vs Envoy" would belong to the Technology tag.
+- **Release**: Tag for release notes. Note that the release notes in blog posts are polished where as inline release notes are written by developers.
+- **Security**: Security vulnerability notifications and methods to bypass security vulnerabilities. Currently there are [five articles](/blog/tags/security/), and they generally have CVE-xxxxxxx in its title.
+- **Technology**: Technical articles. Should not be confused with **Practical Case** (see above).
+- **User Case**: Posts about using Apache APISIX. Tell us how you are using Apache APISIX!
 
-1. Open Chrome or your preferred browser.
-2. Enter the author's GitHub address to access the author's GitHub home page.
+Reviewers will help you find the right tags while reviewing your PR.
+##### Obtaining authors.image_url
+
+1. Open your browser.
+2. Enter the url to author's GitHub page (https://github.com/author-username).
 3. Right click on the author's avatar, and click "copy image address".
-4. Paste image address to the authors.image_url field.
+4. Paste image address to the `authors.image_url` field.
+
 ![How to get the authors.image_url](https://static.apiseven.com/202108/1638257037354-10fc1d17-c0f7-4826-84c9-5966b1ae43ad.png)
```

Comment:
It would be nice to change the example picture to something general. Maybe this work https://github.com/octocat. Or may not need it at all if we change it to the above method.

Reply:
Do I need access to host images on the CDN? I can easily record a GIF and add it here if it would be better.